product_docs/docs/pgd/4/deployments/tpaexec/using_tpaexec.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Using TPAexec
+
+---
+
+With TPAexec you configure, provision, and deploy your EDB Postgres Distributed clusters.
+
+## Configure
+The `tpaexec configure` command generates a simple YAML configuration file to describe a cluster, based on the options you select. The configuration is ready for immediate use and you can modify it to better suit your needs. Editing the configuration file is the usual way to make any configuration changes to your cluster both before and after it's created.
+
+The syntax is:
+
+```
+tpaexec configure <cluster_dir> --architecture <architecture_name> --layout <layout> [options]
+```
+
+The required and recommended configuration options include: 
+
+| Flags                        | Description |
+| ---------------------------- | ----------- |
+| `--architecture`             | Required. Set to `BDR-Always-ON` for EDB Postgres Distributed deployments. |
+| `--layout`                   | Required. Specify one of the four supported architectures: bronze, silver, gold, and platinum. See [Choosing your architecture](/pgd/4/architectures) for more information. |
+| `--harp-consensus-protocol`  | Required. Use `bdr` if your layout is bronze or silver. Use `etcd` if your layout is gold or platinum.              |
+| `--enable-camo`              | Optional. Set the pair of BDR primary instances in each region to be each other's CAMO partners. |
+
+For example:
+
+```
+[tpa]$ tpaexec configure ~/clusters/speedy \
+         --architecture BDR-Always-ON \
+         --layout gold \
+         --harp-consensus-protocol etcd \
+         --enable-camo  
```

Comment:
I'd skip the enable-camo, worried peoples will just copy paste without realizing implications of CAMO